protocol/v2/ssv/validator/domain_cache.go

```diff
@@ -0,0 +1,50 @@
+package validator
+
+import (
+	"time"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/jellydator/ttlcache/v3"
+
+	"github.com/ssvlabs/ssv/protocol/v2/blockchain/beacon"
+)
+
+type DomainCache struct {
```

Comment:
nice addition, we should probably use this in the actual runners and not just exporter! of course in a different PR.